manifest.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
should this file be deleted for the OS repo?

Reply:
Yes, I suppose!